client/packages/system/src/Patient/Components/ChipTableCell.tsx

```diff
@@ -23,6 +23,8 @@ export const ChipTableCell = <T extends RecordWithId>({
         alignItems: 'center',
         display: 'flex',
         flexWrap: 'wrap',
+        gap: 0.2,
+        padding: '3px 0',
```

Comment:
It's too close together and really doesn't look nice... so have add a little spacing 🙈